### ISSUE TITLE
Change Accept Header

### ISSUE THREAD

**Issue by Moon-Hub** [KnpLabs/php-github-api#680]
Like Issue #507 , working with the code-search, i want to change my accept header to the text-match header `application/vnd.github.v3.text-match+json`

Following along from the example in 507, my code:
```
// CONNECT TO API AND AUTHENTICATE SELF
$builder = new Github\HttpClient\Builder(new GuzzleClient());
$client = new \Github\Client($builder, 'application/vnd.github.v3.text-match+json');
$client->authenticate($user, $pwd, Github\Client::AUTH_HTTP_PASSWORD);
```

But, i'm greeted with: `Fatal error: Uncaught Error: Class 'GuzzleClient' not found`
So, what's wrong here?

**Comment by acrobat** [comment]
`GuzzleClient` is not an existing class but as alias. Add this to your uses `use Http\Adapter\Guzzle6\Client as GuzzleClient;`

We should also update the docs so the correct example code is used.

**Comment by Moon-Hub** [comment]
Funny, i was just about to post that i got my text-match to show but there was an issue with the builder, the `use Http\Adapter\Guzzle6\Client as GuzzleClient` has fixed it completely!

**Comment by acrobat** [comment]
I've fixed the docs in #679
